opencti-platform/opencti-front/src/private/components/analyses/reports/ReportKnowledgeGraphBar.jsx

```diff
@@ -385,6 +385,7 @@ class ReportKnowledgeGraphBar extends Component {
         && numberOfSelectedNodes === 0
         && numberOfSelectedLinks === 1
         && selectedLinks.length === 1
+        && selectedLinks[0].entity_type !== 'basic-relationship'
```

Comment:
Only in correlation view

Reply:
the KnowledgeGraphBar is used for all the views in the Report Knowledge tab... so also for the Correlation view